listenbrainz/webserver/static/js/__tests__/lastFmImporter.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+import LastFmImporter from '../jsx/lastFmImporter'
+
+let props;
+
+describe('LastFmImporter is working correctly', () => {
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<LastFmImporter {...props} />, div);
```

Comment:
why are we using ReactDom here? 

Ideally I'd like to use enzyme to test the components, something liek mounting the component and checking that it is truthy. https://airbnb.io/enzyme/docs/api/mount.html

```
it ('renders', () => {
    const wrapper = mount(<LastFmImporter {...props />);
    expect(wrapper).toBeTruthy();
})
```

Something around these lines.